### PR TITLE
Feat: add some documents about the user and RBAC

### PR DESCRIPTION
**Pull request by barnettZQG**
Signed-off-by: barnettZQG <barnett.zqg@gmail.com>

### CODE REVIEW

File: docs/how-to/dashboard/user/project.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Project management
+---
+
+Projects provide a logical grouping of applications、environments and targets, which is useful when VelaUX is used by multiple teams. Projects provide the following features:
```
Comment (wonderflow) [inline docs/how-to/dashboard/user/project.md]:
```suggestion
Project provides a logical separation of applications、environments and delivery targets, this is helpful when VelaUX is used by different teams. Project can provide the following features:
```

File: docs/how-to/dashboard/user/project.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+* restrict where apps may be deployed to (target clusters and namespaces), we plan to support setting resource limits for the target.
```
Comment (wonderflow) [inline docs/how-to/dashboard/user/project.md]:
```suggestion
* applications can be deployed to the target clusters and namespaces belong to the same project.
```

没实现的就先不写上去了。

File: docs/how-to/dashboard/user/project.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+* restrict what Integrations can be use(trusted Helm repositories, docker registry or cloud provider)
```
Comment (wonderflow) [inline docs/how-to/dashboard/user/project.md]:
```suggestion
* restrict which kinds of integration can be use, you can specify config within a project that contains trusted Helm repositories, docker registries or cloud providers.
```

注意单复数，和末尾句号。

File: docs/how-to/dashboard/user/project.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+* defining project roles to provide application RBAC.
```
Comment (wonderflow) [inline docs/how-to/dashboard/user/project.md]:
```suggestion
* project level RBAC, the project administrator can control the application level authorization.
```

File: docs/how-to/dashboard/user/project.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+* restrict what kinds of objects or component definitions may or may not be deployed(TODO)
```
Comment (wonderflow) [inline docs/how-to/dashboard/user/project.md]:
```suggestion
```

没实现的没必要写到文档，写到Roadmap去。

File: docs/how-to/dashboard/user/project.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+## The Default Project
+
+Once installed VelaUX has one default project with name `Default`, and owner by admin user. At the same time, the default target and environment are also generated by default and bound in the project.
```
Comment (wonderflow) [inline docs/how-to/dashboard/user/project.md]:
```suggestion
Once installed VelaUX has one default project with name `Default` owned by the `admin` user. At the same time, the default target and environment are also generated automatically and bound to the project.
```

File: docs/how-to/dashboard/user/project.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+## Creating Projects
+
+Users with project management rights can go to `Platform/Projects` page，in this page you could create and delete project.
```
Comment (wonderflow) [inline docs/how-to/dashboard/user/project.md]:
```suggestion
Users with project management permissions can go to `Platform/Projects` page for settings，in this page you could create and delete project.
```

File: docs/how-to/dashboard/user/project.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+
+![project-list](https://static.kubevela.net/images/1.3/project-list.jpg)
+
+Click the `New Project` button, you can open the drawer page for creating a project. Each project should set a name and owner, the owner user can auto grant the project admin role after the project is created.
```
Comment (wonderflow) [inline docs/how-to/dashboard/user/project.md]:
```suggestion
Click the `New Project` button for creating a project. Each project should set name and owner, the owner user will be granted the project admin role automatically after the project created.
```

File: docs/how-to/dashboard/user/project.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+
+## Updating Projects
+
+Project owner, alias, and description information support updating. Click the project name and go to the project detail page, you can manage the members and roles in this project.
```
Comment (wonderflow) [inline docs/how-to/dashboard/user/project.md]:
```suggestion
Project owner, alias, and description fields can be updated. Click the project name and go to the project detail page, you can manage the members and roles in this project.
```

File: docs/how-to/dashboard/user/project.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+
+## Deleting Projects
+
+If there are applications and environments in a project, it can not be deleted. you must delete applications, environments, and targets in the project that you want to delete.
```
Comment (wonderflow) [inline docs/how-to/dashboard/user/project.md]:
```suggestion
If there are applications and environments living in one project, then deleting the project is not allowed. You must clean up the applications, environments, and targets in the project before you can delete.
```

File: docs/how-to/dashboard/user/user.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
+---
+title: User management
+---
+
+Once installed VelaUX has one built-in `admin` user that has full access to the system. it is recommended to use this user only for initial configuration and then create new user or configure SSO integration.
```
Comment (FogDong) [inline docs/how-to/dashboard/user/user.md]:
```suggestion
Once VelaUX is installed, there's a built-in administrator user with full access to the system. It is recommended to use only the admin user for the initial configuration, then switch to another user or configure the SSO integration.
```

File: docs/how-to/dashboard/user/user.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+## Local users/accounts
+
+The local users/accounts feature serves two main use-cases:
+
+* For the restful API users，it is possible to configure an API account with limited permissions and generate an authentication token. users can use token to communicate with the API to create applications, etc.
+* For a very small team where use of SSO integration might be considered an overkill. There could only use local users to authentication. 
+
+### Get full users list
+
+If login user have list users permission, he could switch to the `Platform/Users` page and get full users list.
```
Comment (FogDong) [inline docs/how-to/dashboard/user/user.md]:
```suggestion
If the logged-in user has the list user permission, he can switch to the `Platforms/Users` page and get the full list of users.```

File: docs/how-to/dashboard/user/project.md (hunk 2 above)
Comment (barnettZQG) [reply docs/how-to/dashboard/user/project.md]: restrict where applications may be deployed to (target clusters and namespaces).

这个表达我认为更准确。

File: docs/how-to/dashboard/user/project.md (hunk 3 above)
Comment (barnettZQG) [reply docs/how-to/dashboard/user/project.md]: restrict what integration configs can be used (trusted Helm repositories, docker registries or cloud providers).

File: docs/how-to/dashboard/user/project.md (hunk 4 above)
Comment (barnettZQG) [reply docs/how-to/dashboard/user/project.md]: restrict the different permission of each user, and provide a multiple-level permission control (RBAC).

File: docs/how-to/dashboard/user/project.md [hunk 13]
```diff
@@ -0,0 +1,29 @@
+---
+title: Project management
+---
+
+Project provides a logical separation of applications、environments and delivery targets, this is helpful when VelaUX is used by multiple teams. Project can provide the following features:
+
+* restrict where applications may be deployed to (target clusters and namespaces).
+* restrict what integration configs can be used (trusted Helm repositories, docker registries or cloud providers).
+* restrict the different permission of each user, and provide a multiple-level permission control (RBAC).
```
Comment (wonderflow) [inline docs/how-to/dashboard/user/project.md]:
```suggestion
* restrict where applications can be deployed to (target clusters and namespaces).
* restrict what kinds of integrated configration can be used (trusted Helm repositories, docker registries or cloud providers).
* restrict the different permission of each user, and provide a multiple-level permission control (RBAC).
```
```suggestion
* restrict where applications may be deployed to (target clusters and namespaces).
* restrict what integration configs can be used (trusted Helm repositories, docker registries or cloud providers).
* restrict the different permission of each user, and provide a multiple-level permission control (RBAC).
```

Comment (wonderflow) [inline docs/how-to/dashboard/user/project.md]:
```suggestion
* restrict where applications can be deployed to (target clusters and namespaces).
* restrict what kinds of integrated configration can be used (trusted Helm repositories, docker registries or cloud providers).
* restrict the different permission of each user, and provide a multiple-level permission control (RBAC).
```
```suggestion
* restrict where applications may be deployed to (target clusters and namespaces).
* restrict what integration configs can be used (trusted Helm repositories, docker registries or cloud providers).
* restrict the different permission of each user, and provide a multiple-level permission control (RBAC).
```

**Review by wonderflow — APPROVED** [review]

**Pull request merged**